docs/WeakMap.md

```diff
@@ -0,0 +1,8 @@
+# WeakMap
```

Comment:
There's not much here -- it seems like it could just go in the top-level readme.